src/st_stuff.c

```diff
@@ -235,7 +235,7 @@ static patch_t *faceback[MAXPLAYERS]; // killough 3/7/98: make array
  // main bar right
 static patch_t *armsbg;
 
-// Bezel bottom edge for st_solidbackground.
+// Bezel bottom edge for widescreen status bar.
 static patch_t *bezel;
```

Comment:
Technically, `PU_CACHE` is lower than `PU_LEVEL`, so theoretically memory could be freed before `bezel` is used again. In our codebase, this will probably never happen, and currently `StatusBarBufferHeight()` and `ST_Warnings()` are only called at initialization with no other dynamic memory allocations. But this may be changed in the future.
Sorry to be pedantic, but for me global variables are always `PU_STATIC` and local variables `PU_CACHE`. I think we should get rid of the global variable `bezel`.

Reply:
No, you're right, that makes sense. :thumbsup: